frontend/elements/src/core/buttons/arrow-button.ts

```diff
@@ -0,0 +1,59 @@
+import { LitElement, html, css, customElement, property } from "lit-element";
+ import { BaseButton } from "@elements/_styles/buttons";
+import {nothing} from "lit-html";
+
+export type Color = "yellow" | "pink"  ;
+export type Direction = "right" | "left"  ;
+ 
+
+@customElement("arrow-button")
+export class _ extends BaseButton {
+  static get styles() {
+    return [
+      css`
+
+      button{
+        width: 48px;
+        height: 48px;
+        cursor:pointer;
+      }
+  
+ 
+ .right{
+
+ }
+
+ .left{
```

Comment:
this is not dynamic. Why scale?

Reply:
cecause i am using with one img and change it according the direction.

Reply:
scale is used when changing size, not direction. use transform: rotate(180deg)